src/components/sidebar/menu-buttons.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classnames from 'classnames';
+import Icon from '@quantumblack/kedro-ui/lib/components/icon';
+import MenuIcon from '../icons/menu';
+
+/**
+ * Hamburger menu button
+ * @param {Function} props.onToggle Show menu on click
+ * @param {Boolean} props.visible Whether nav is visible
+ */
+export const ShowMenuButton = ({ onToggle, visible }) => (
+  <button
+    aria-label="Show menu"
+    className={classnames(
+      'pipeline-sidebar__show-menu pipeline-sidebar__icon-button',
+      {
+        'pipeline-sidebar__icon-button--visible': !visible
```

Comment:
Do we have to pass `visible` in here? `onToggle` is already hard-coded to `true` right. The same applies to `HideMenuButton`. The reason I'm asking is because `'pipeline-sidebar__icon-button--visible': !visible` is a bit hard to parse for my brain, but it's not a biggie.

Reply:
Yeah that's a really tricky one - I'm using 'visible' to refer to two different things - the visibility of the sidebar, and the visibility of the Show Menu button, which is only visible when the other is hidden. I think renaming the prop to be more specific will help - I'll do that. Thanks!